api/src/main/java/org/openmrs/validator/AlertValidator.java

```diff
@@ -41,13 +47,16 @@ public boolean supports(Class<?> c) {
 	 * @should fail validation if field lengths are not correct
 	 */
 	@Override
-	public void validate(Object obj, Errors errors) {
+	public void validate(Object obj, Errors errors) throws APIException {
+		log.debug("{}.validate...", this.getClass().getName());
+
+		if (obj == null || !(obj instanceof Alert)) {
```

Comment:
Better to write unit tests for these conditions

Reply:
This passes the unit test set

Reply:
Is there a unit test to validate the exception when obj is null or its not a instance of Alert ?

Reply:
I will take a look at this have been busy with some shool work lately

Reply:
The is a unit test this.